diff --git a/config.def.h b/config.def.h
index 1c0b587..38d2f6c 100644

--- config.def.h
+++ config.def.h
@@ -45,7 +45,7 @@
 };
 
 /* key definitions */
-#define MODKEY Mod1Mask
+#define MODKEY Mod4Mask
 #define TAGKEYS(KEY,TAG) \
 	{ MODKEY,                       KEY,      view,           {.ui = 1 << TAG} }, \
 	{ MODKEY|ControlMask,           KEY,      toggleview,     {.ui = 1 << TAG} }, \
 
-- 
2.20.1

